deploy/helm/cht-chart/values.yaml

```diff
@@ -0,0 +1,34 @@
+project_name: <your-project-name>
```

Comment:
I know I have found it particularly useful to be able to run this PR with a default set of values, as defined here. But helm execution breaks because of these params with angled brackets. To help future users of this script, would it be worth changing these default params in angled params to valid helm params (e.g. `<your-project-name>` to a valid release name like `test-project`) so that future SREs get stuff running, hopefully at the first go and refactor these params to their desire later.